src/main/java/edu/harvard/hms/dbmi/bd2k/irct/controller/ExecutionController.java

```diff
@@ -213,6 +213,7 @@ public Result call() {
 					result.setResultStatus(ResultStatus.ERROR);
 					result.setMessage(e.getMessage());
 				} catch (Exception e) {
+					e.printStackTrace();
```

Comment:
nitpicking here but it's generally not good practice to print the stack trace. Are we doing this regularly so it gets sent to Splunk?